Cargo.toml

```diff
@@ -26,6 +26,7 @@ members = [
     "pgx-examples/strings",
     "pgx-examples/triggers",
     "pgx-examples/versioned_so",
```

Comment:
Could you also add a change to `.github/workflows` as well? https://github.com/tcdi/pgx/blob/1a89778046547c52c714fe6885069eab5638f965/.github/workflows/tests.yml#L180-L181